### PR TITLE
Add parallel place startup feature

### PR DESCRIPTION
**Pull request by fbruton**

### CODE REVIEW

File: src/main/config/node.cfg [hunk 1]
```diff
@@ -2,4 +2,6 @@
 URL = "@{emissary.node.scheme}://@{emissary.node.name}:@{emissary.node.port}"
 LOCAL_DIRECTORY = "@{URL}/DirectoryPlace"
 
+PARALELL_PLACE_STARTUP = "false"
```
Comment (ukengineer925) [inline src/main/config/node.cfg:5]:
```suggestion
PARALLEL_PLACE_STARTUP = "false"
```

File: src/main/java/emissary/admin/Startup.java [hunk 2]
```diff
@@ -25,6 +28,7 @@
     public static final int DIRECTORYSTART = 0;
     public static final int DIRECTORYADD = 1;
     public static final int DIRECTORYDELETE = 2;
+    private static final String PARALELL_PLACE_STARTUP_CONFIG = "PARALELL_PLACE_STARTUP";
```
Comment (ukengineer925) [inline src/main/java/emissary/admin/Startup.java:31]:
```suggestion
    private static final String PARALLEL_PLACE_STARTUP_CONFIG = "PARALLEL_PLACE_STARTUP";
```

File: src/main/java/emissary/admin/Startup.java [hunk 3]
```diff
@@ -342,27 +346,30 @@ public void run() {
 
                 logger.info("Using localDir={} to create {} places on {}", localDirectory, hostParameters.size(), thePlaceHost);
 
+                // Create a stream of places that can be configured to start in parallel
+                boolean paralellPlaceStartup = hostsConfig.findBooleanEntry(PARALELL_PLACE_STARTUP_CONFIG, false);
+                Stream<String> hostParametersStream = StreamSupport.stream(hostParameters.spliterator(), paralellPlaceStartup);
```
Comment (ukengineer925) [inline src/main/java/emissary/admin/Startup.java:351]:
```suggestion
                boolean parallelPlaceStartup = hostsConfig.findBooleanEntry(PARALLEL_PLACE_STARTUP_CONFIG, false);
                Stream<String> hostParametersStream = StreamSupport.stream(hostParameters.spliterator(), parallelPlaceStartup);
```

**Review by ukengineer925 — APPROVED** [review]
LGTM - some spelling tweaks within.

**Review by jpdahlke — DISMISSED** [review]
Please address spelling changes

**Review by dev-mlb — APPROVED** [review]

**Pull request merged**